feature-libs/asm/occ/adapters/occ-asm.adapter.ts

```diff
@@ -65,4 +66,27 @@ export class OccAsmAdapter implements AsmAdapter {
       .get<CustomerSearchPage>(url, { headers, params })
       .pipe(this.converterService.pipeable(CUSTOMER_SEARCH_PAGE_NORMALIZER));
   }
+
+  bindCart({ cartId, customerId }: BindCartOptions): Observable<void> {
```

Comment:
Return type should probably be `Observable<unknown>`  to be consistent with the rest